qemu/tests/cfg/fwcfg.cfg

```diff
@@ -19,21 +19,17 @@
     image_name_stg = "images/storage"
     remove_image_stg = yes
     no ppc64 ppc64le aarch64
+    chk_windbg_cmd = 'powershell -Command "Get-WmiObject Win32_Product | Where-Object Name -like '*SDK Debuggers*' | Select Name, Version"'
```

Comment:
Hi @heywji , just to confirm whether all win10+ guests have the capability to use powershell by default?

Reply:
Yes, I test all the Windows NT versions (both personal and server versions). All can be run it successfully.

Reply:
Thanks, if win 10+ version works, it looks good to me.